docs/wire_grpc.md

```diff
@@ -137,12 +138,14 @@ interface RouteGuideServer : Service {
 The server can use Kotlin coroutines' suspend and Channels mechanisms to execute suspending network
 calls.
 
-### Sample
+Sample
+------
 
 Check out our [sample project][sampleProject] demonstrating bidirectional communication between an
 Android application and a server.
 
-### Get it
+Get it
+------
```

Comment:
This will change from an h3 tag to an h2 tag, making them into their own sections instead of being listed under "gRPC for Servers".  is that what you want?

Reply:
yes